task2/src/server.js

```diff
@@ -1,18 +1,29 @@
 import express from 'express'
 import router from './router.js'
 
-const server = express()
-server.use('/', router)
+const startServer = () => {
+      const server = express()
+      server.use('/', router)
+      server.use(function (error, req, response, next){
+            if (error.type === 'entity.parse.failed') {
+                  response.status(400).send('Not valid body in request.')
+            } else {
+                  response.status(400).send(`Error in request:\n${error}`)
+            }
+            next()
```

Comment:
If you don't have a specific use case after handling error, writing next() function will allow express to run any other middleware if there. Also above next() if you write any code it runs as well. You can reference here https://stackoverflow.com/a/16180550